pando/http/request.py

```diff
@@ -71,27 +69,12 @@ def make_franken_uri(path, qs):
 
     """
     if path:
-        try:
-            path.decode('ASCII')    # NB: We throw away this unicode!
-        except UnicodeDecodeError:
-
-            # XXX How would we get non-ASCII here? The lookout.net post
-            # indicates that all browsers send ASCII for the path.
-
-            # Some servers (gevent) clobber %2F inside of paths, such
-            # that we see /foo%2Fbar/ as /foo/bar/. The %2F is lost to us.
-            parts = [quote(x) for x in quoted_slash_re.split(path)]
-            path = b"%2F".join(parts)
+        # Some servers (gevent) clobber %2F inside of paths, such
+        # that we see /foo%2Fbar/ as /foo/bar/. The %2F is lost to us.
+        path = quote(path).encode('ascii')
```

Comment:
On second thought this is wrong because properly escaped paths would be escaped a second time. Passing '%' as the second argument to `quote` will avoid this.